src/software/jetson_nano/services/network.cpp

```diff
@@ -30,12 +30,33 @@ std::tuple<TbotsProto::PrimitiveSet, TbotsProto::World> NetworkService::poll(
                                                                    world_msg};
 }
 
+float calculatePacketLossRate(std::queue<uint64_t> recent_proto_seq_nums,
+                              uint64_t seq_num)
+{
+    recent_proto_seq_nums.push(seq_num);
+
+    // Pop sequence numbers that are no longer recent
+    while (seq_num - recent_proto_seq_nums.front() >= RECENT_PACKET_LOSS_PERIOD)
+    {
+        recent_proto_seq_nums.pop();
+    }
+
+    // seq_num + 1 is to account for the sequence numbers starting from 0
```

Comment:
This comment (`+1`) should probably be reverted to reflect the removal of the +1